src/public/parts/shells/serviceSourceManage/public/index.html

```
@@ -29,9 +29,9 @@ <h1 class="w-full text-3xl font-bold mb-4" data-i18n="serviceSource_manager.titl
 		<div class="w-full md:w-1/4 mb-4 flex flex-col gap-3">
 			<div class="flex justify-between items-center m-2">
 				<h2 class="text-xl font-semibold" data-i18n="serviceSource_manager.fileList.title"></h2>
-				<button id="addFileButton" class="btn btn-sm btn-circle btn-primary" data-i18n="serviceSource_manager.fileList.addButton.title">+</button>
+				<button id="addFileButton" class="btn btn-sm btn-square btn-primary" data-i18n="serviceSource_manager.fileList.addButton.title">+</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)